api/lib/opentelemetry/correlation_context/manager.rb

```diff
@@ -24,6 +25,10 @@ def value(key, context: Context.current)
         nil
       end
 
+      def values(context: Context.current)
```

Comment:
I called this `values` to line up with `set_value`, `value`, `remove_value`, but `correlations` may be more appropriate.

WDYT @mwear?

Reply:
I think `values` is the perfect choice. The correlation context manager is accessible through the `OpenTelemetry` module as `OpenTelemetry.correlations`. So, this would end up being `OpenTelemetry.correlations.values`, which is much better than `OpenTelemetry.correlations.correlations`.